docs/reference/zedtokens-and-zookies.md

```diff
@@ -136,54 +137,14 @@ You can always get a new one by issuing any call in v1 (`CheckPermission` with [
 
 The SpiceDB API will default to [Consistency] of `minimize_latency`.
 
-If you do not intend to store ZedTokens, it is recommended that **all** calls use [Consistency] of `fully_consistent` (which has a major performance hit, but is fully safe). Really, though, you should store ZedTokens.
-
-:::warning
-Only ever using a consistency of `minimize_latency` can expose your application up to the [New Enemy Problem].
-:::
-
-## Internal Implementation Details
-
-In SpiceDB, a ZedToken is the v1 equivalent of the v0 Zookie and are, in fact, *wire back compatible* with Zookies to allow for ease of upgrade.
-
-- Zed Tokens (v1):
-  - [External Proto Definition](https://buf.build/authzed/api/file/main/authzed/api/v1/core.proto#L53)
-  - [Documentation](https://pkg.go.dev/github.com/authzed/spicedb/pkg/zedtoken)
-  - [Implementation](https://github.com/authzed/spicedb/tree/main/pkg/zedtoken)
-- Zookies (v0):
-  - [External Proto Definition](https://buf.build/authzed/api/file/main/authzed/api/v0/core.proto#L60)
-  - [Documentation](https://pkg.go.dev/github.com/authzed/spicedb/pkg/zookie)
-  - [Implementation](https://github.com/authzed/spicedb/tree/main/pkg/zookie)
-
-[➡️ Proto Definitions for the encoded forms](https://github.com/authzed/spicedb/tree/main/proto/internal/impl/v1/impl.proto)
+If you do not intend to store ZedTokens and you care about the New Enemy Problem, it is recommended that **all** calls use [Consistency] of `fully_consistent` (which has a major performance hit, but is fully safe). Really, though, you should store ZedTokens.
 
-### Zookie vs snapshot tokens
+#### But I really don't want to store a ZedToken
```

Comment:
Rather than having the title be about storing zedtokens maybe it should be about tolerating the new enemy problem

Reply:
People won't read it otherwise; they really want to know what happens if they just ignore it